frontend/tests/controllers/TeamGroupsTest.php

```diff
@@ -587,4 +587,104 @@ public function testRemoveTeamsFromTeamsGroup() {
 
         $this->assertEmpty($teamIdentities);
     }
+
+    public function testSwitchBetweenAssociatedIdentities() {
+        // Identity creator group member will upload csv file
+        [
+            'identity' => $creatorIdentity,
+        ] = \OmegaUp\Test\Factories\User::createGroupIdentityCreator();
+        $creatorLogin = self::login($creatorIdentity);
+        [
+            'teamGroup' => $teamGroup,
+        ] = \OmegaUp\Test\Factories\Groups::createTeamsGroup(
+            $creatorIdentity,
+            null,
+            null,
+            null,
+            $creatorLogin
+        );
+        $numberOfUsers = 10;
```

Comment:
```suggestion
        $numberOfUsers = 2;
```

no es necesario hacer tantas rondas.